garden-backend-service/src/api/routes/gardens.py

```diff
@@ -67,13 +78,17 @@ async def delete_garden(
 @router.put("/{doi:path}", response_model=GardenMetadataResponse)
 async def create_or_replace_garden(
     doi: str,
+    background_tasks: BackgroundTasks,
     garden_data: GardenCreateRequest,
     db: AsyncSession = Depends(get_db_session),
     user: User = Depends(authed_user),
+    settings: Settings = Depends(get_settings),
 ):
     existing_garden: Garden | None = await Garden.get(db, doi=doi)
     if existing_garden is None:
-        return await _create_new_garden(garden_data, db, user)
+        return await _create_new_garden(
```

Comment:
this smells a little off to me, I think that `_create_new_garden` shouldn't also be responsible for sending off background tasks. imo we want to dispatch background tasks from the routes themselves rather than from inside of utility functions